src/libstd/time/mod.rs

```diff
@@ -140,7 +140,7 @@ impl SystemTime {
     /// guaranteed to always be before later measurements (due to anomalies such
     /// as the system clock being adjusted either forwards or backwards).
     ///
-    /// If successful, `Ok(duration)` is returned where the duration represents
+    /// If successful, `Ok(Duration)` is returned where the duration represents
```

Comment:
There is no variable named `duration` anywhere here, so I felt using a type name would be more suitable. It would also be implied that we are referring to the type of the returned value.